### DISCUSSION TITLE
Missing Tabs after switching to 3d-printer

### DISCUSSION THREAD

**Discussion by andreasthaler1966** [synman/Octoprint-Bettergrblsupport#84] [Q&A]
Hi!
I just installed the GREAT plugin on my PI, wich I normally use for a 3D-Printer.
Now I wanted to switch back to the 3D-Printer (by disabeling the plugin).
Now my #control, #temp and  #gcode - tabs disappeared. 
I hope this is the correct channel to question...

**Comment by synman** [comment]
you have to uninstall the plugin for it to revert its system level changes.  there is no hook I can tie into when you disable a plugin, unfortunately.
